### PR TITLE
Add sury/php package repository

### PR DESCRIPTION
**Pull request by chalasr**
Adding https://packages.sury.org/php for using PHP 7.1.
Let me know if tests need to be updated or something else (travis build is failing but it does not seem to be related to this change, httpredir bothers).

### CODE REVIEW

**Comment by nervo** [comment]
Issues are not related to httpredir itself but to an ansible 2.2.1 bug https://github.com/ansible/ansible/issues/20885
I'm looking for a workaround
I'm tired...

File: vars/main.yml [hunk 1]
```diff
@@ -230,10 +230,10 @@ manala_apt_keys_patterns:
     id:  86E50310
   sury_php_debian:
     url: https://packages.sury.org/php/apt.gpg
-    id: 4A7A714D
+    id:  4A7A714D
   sury_php_ubuntu:
     keyserver: hkp://keyserver.ubuntu.com:80
-    id: E5267A6C
+    id:        E5267A6C
```
Comment (chalasr) [inline vars/main.yml]: Yet I agree about aligning them all, I think it should be consistent all along the file (see e.g. https://github.com/manala/ansible-role-apt/pull/15/commits/f1f6e33b0ee9cfb3ff27937aa6f8f217cabdc914#diff-6d62b73b3f46cfa11f6466d08cf45303R215)

Comment (nervo) [reply vars/main.yml]: It's aligned with the content of the upper key, wich could be either "url" or "keyserver".
Or do i miss something ?

Comment (chalasr) [reply vars/main.yml]: Ok got it, strange approach though (to me it's either _all values_ are aligned with all others or not at all, not per entry indent).

But following your logic, I can still find some incorrect ones :) https://github.com/manala/ansible-role-apt/pull/15/files/edd094fd88a79a112b0d1c099ae5e7f9321e6e38..f1f6e33b0ee9cfb3ff27937aa6f8f217cabdc914#diff-6d62b73b3f46cfa11f6466d08cf45303R132

Comment (nervo) [reply vars/main.yml]: How do you say relou in english ? :)

Comment (chalasr) [reply vars/main.yml]: I say: thanks for the fixes @nervo

**Pull request merged**